Description: systemd: Support dhcp
 Debian has an alternate /var/lib/ntp/ntp.conf.dhcp configuration file for
 integration with dhclient.
Forwarded: not-needed
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2017-11-06
--- a/etc/ntpd.service
+++ b/etc/ntpd.service
@@ -13,7 +13,7 @@
 # the clock on boot.  However, if Restart=yes is set, a malicious (or broken)
 # server could send the incorrect time, trip the panic threshold, and when
 # ntpd restarts, serve it the incorrect time (which would be accepted).
-ExecStart=/usr/sbin/ntpd -n -g -N -u ntp:ntp
+ExecStart=/bin/sh -c 'exec /usr/sbin/ntpd -n -g -N -u ntp:ntp $(if [ /var/lib/ntp/ntp.conf.dhcp -nt /etc/ntp.conf ] ; then echo "-c /var/lib/ntp/ntp.conf.dhcp" ; fi)'
 Restart=no
 
 [Install]
